### PR TITLE
AdHash Bid Adapter: brand safety changes

### PR DESCRIPTION
**Pull request by wyand-sp**
Changing the recent ads calculations to the latest supported version.
Removed the usage of body.innerText

## Type of change
- [x] Feature
- [x] Build related changes

## Description of change
Changing the recent ads calculations to the latest supported version.
Removed the usage of body.innerText as needed in Prebid JS 9.0

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@wyand-sp this test failure should be fixed in the latest version of master if you want to pull in recent commits

File: modules/adhashBidAdapter.js [hunk 1]
```diff
@@ -120,7 +121,7 @@ function brandSafety(badWords, maxScore) {
       .replaceAll(/\s\s+/g, ' ')
       .toLowerCase()
       .trim();
-    const content = window.top.document.body.innerText.toLowerCase();
+    const content = window.top.document.body.outerText.toLowerCase();
```
Comment (patmmccann) [inline modules/adhashBidAdapter.js:124]: does textContent not work? that's what we used in 9.0 branch

Comment (patmmccann) [reply modules/adhashBidAdapter.js:124]: https://kellegous.com/j/2013/01/26/layout-performance/ outerText just as bad

Comment (wyand-sp) [reply modules/adhashBidAdapter.js]: Understood. We've updated the code. Thanks for your comment.

**Review by jsnellbaker — APPROVED** [review]

**Pull request merged**